### ISSUE TITLE
Sidecar works incorrectly

### ISSUE THREAD

**Issue by YuHaotian-Tongji** [SchmockLord/Hackintosh-Intel-i9-10900k-Gigabyte-Z490-Vision-D#16]
First,think you for your efi.I'm just run my hackintosh.As the title, the sidecar function seems can't work.I can find the sidecar coin in the dock and so does my ipad pro.This function are used less for me,so you can forget it.

**Comment by SchmockLord** [comment]
What WiFi card do you have?

**Comment by YuHaotian-Tongji** [comment]
BCM94360CD, same as yours

**Comment by YuHaotian-Tongji** [comment]
And I can use sidecar before I changed my mainboard from MSI B360M to Gigabyte z490 vision D

**Comment by SchmockLord** [comment]
How about Airdrop, is that working for you? I haven't tried Sidecar yet, but it showed my iPad Pro in the settings. Will try that tomorrow.

**Comment by YuHaotian-Tongji** [comment]
Airdrop is nice，look forward to your good news^_^

**Comment by SchmockLord** [comment]
I got Sidecar working in my iMac19,1 config.plist.

I am using these settings for my iGPU:

```
<dict>
	<key>PciRoot(0x0)/Pci(0x2,0x0)</key>
	<dict>
		<key>AAPL,ig-platform-id</key>
		<data>AACbPg==</data>
		<key>device-id</key>
		<data>kj4AAA==</data>
		<key>framebuffer-patch-enable</key>
		<data>AQAAAA==</data>
		<key>model</key>
		<string>Intel UHD Graphics 630 (Desktop)</string>
	</dict>
</dict>
```

**Comment by sebastienb** [comment]
@SchmockLord besides the device properties did you change any bios settings to get it to work? I changed the devices properties on my end but I still get a black screen on ipad when it connects.

**Comment by SchmockLord** [comment]
Are you using iMac19,1? iGPU enabled in the BIOS? Native WiFi and Bluetooth support?

**Comment by Scope666** [comment]
I tried your exact iGPU definition, but Sidecar isn't working with my iPad Air 2.  Everything else works including Airdrop, Handoff, Continuity, phone calls, etc.

My board is an Asus Strix Z490-E instead of yours, not sure if that would make a big difference.

![image](https://user-images.githubusercontent.com/31927547/84611589-50a70900-ae8c-11ea-9007-5712aa65d679.png)

**Comment by sebastienb** [comment]
> Are you using iMac19,1? iGPU enabled in the BIOS? Native WiFi and Bluetooth support?
> 
> Maybe this will help you: https://www.reddit.com/r/hackintosh/comments/dwbncg/sidecar_and_smbios_imac191_vs_imacpro11/
> 
> I just don't wanna switch my SMBIOS all the time. I am primarily using iMacPro1,1.

It worked for me. I had to set the gpu to enabled in the BIOs. I initially tried it with Auto. I notice that when it’s on it takes longer to
Boot and also the igpu shows up as Intel unknown in sensei. Sidecar works with it on so I might keep it.

**Comment by Scope666** [comment]
Yeah, I have iGPU enabled in BIOS as well, can see it and the RX 560 both reported in Hackintool above... stumped as to why it won't work.

**Comment by junkyfunk3** [comment]
> I got Sidecar working in my iMac19,1 config.plist.
> 
> I am using these settings for my iGPU:
> 
> ```
> <dict>
> 	<key>PciRoot(0x0)/Pci(0x2,0x0)</key>
> 	<dict>
> 		<key>AAPL,ig-platform-id</key>
> 		<data>AACbPg==</data>
> 		<key>device-id</key>
> 		<data>kj4AAA==</data>
> 		<key>framebuffer-patch-enable</key>
> 		<data>AQAAAA==</data>
> 		<key>model</key>
> 		<string>Intel UHD Graphics 630 (Desktop)</string>
> 	</dict>
> </dict>
> ```
> 
> ![Sidecar-iMac19,1](https://user-images.githubusercontent.com/19785918/84603608-dd9f9180-ae8f-11ea-81dc-c112a607b485.jpg)

Cant thank you enough for posting this.  Worked like a charmed for me as well.  You are awesome

**Comment by junkyfunk3** [comment]
<img width="877" alt="Screen Shot 2020-06-23 at 12 46 21 AM" src="https://user-images.githubusercontent.com/5334901/85361853-0278b200-b4eb-11ea-9913-35900eb3307d.png">

Sidecar working now with iGPU Enabled in BIOS which is so great but wonder if this is causing my Radeon RX580 a memory leak?  iStat is showing the memory slowly increasing even though I'm using the OS fairly lightly.  Any thought?

**Comment by junkyfunk3** [comment]
everything else seems to work with no hiccups. not graphic stuttering or anything noticeable other than this mete going up and up

**Comment by SchmockLord** [comment]
Try disabling the Radeonboost.kext.

**Comment by junkyfunk3** [comment]
Hm,  I don't have that one in my EFi. Perhaps I should inject it?   Also,   I wonder... Perhaps this is happening because I installed the RX580 in the middle PCIe x8 slot?  I put there because the enormous noctua cooler was too big for the GPU to fit in the top slot.  I realized after installing the heatsink that I can rotate  it  so it will fit which which would point the fans toward the back of the case instead of the top.  I think i will prefer this for the future to reduce fan noise coming out of the top of the case.
